lib/t_web/channels/feed_channel.ex

```diff
@@ -140,6 +140,11 @@ defmodule TWeb.FeedChannel do
     {:reply, :ok, socket}
   end
 
+  def handle_in("seen-match", %{"match_id" => match_id}, socket) do
+    Matches.mark_match_seen(me_id(socket), match_id)
```

Comment:
add timings here as well? https://github.com/getsince/test3/pull/599#discussion_r812211848

Reply:
wouldn't it duplicate `seen` timings?

Reply:
No, since different feedProfileItem.kinds have separate seen commands:
![image](https://user-images.githubusercontent.com/16062322/155262576-bf2cd8ff-1712-4693-beb0-83d8457ed5fe.png)